README.md

```diff
@@ -1,28 +1,51 @@
-# Terraform module to reference common tags
-
-## Tagging policy documentation
-
-https://tools.hmcts.net/confluence/display/DCO/Tagging+v0.4
-
+# terraform-module-common-tags 
+
+A Terraform module to reference common tags.
+## [Tagging policy documentation](https://tools.hmcts.net/confluence/display/DCO/Tagging+v0.4)
+
+## Usage
+
+This example shows you how to add  `expiresAfter` tag with a specific expiry date. `expiresAfter` tag only applies to sbox environment. 
```

Comment:
first example should have the default example, expires after should be a separate example